libraries/AP_Compass/AP_Compass_Calibration.cpp

```diff
@@ -528,7 +528,7 @@ MAV_RESULT Compass::mag_cal_fixed_yaw(float yaw_deg, uint8_t compass_mask,
     float inclination;
     if (!AP_Declination::get_mag_field_ef(lat_deg, lon_deg, intensity, declination, inclination)) {
         GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "Mag: WMM table error");
-        return MAV_RESULT_FAILED;
```

Comment:
Can you add static asserts:
static_assert(MAV_RESULT_FAILED == false)
static_assert(MAV_RESULT_ACCEPTED == true)

Reply:
Sorry, no need for this. I need some coffee.